ruma-federation-api/CHANGELOG.md

```diff
@@ -38,6 +39,10 @@ Improvements:
   },
   ```
 
+Bug fixes:
+
+* Properly decode well-known response
```

Comment:
This fix affects serialization as well as deserialization, would be nice to clear that up here.

Reply:
Yes, you are right. Thanks for pointing that out :+1: